consensus/src/pow/constants.rs

```diff
@@ -0,0 +1,14 @@
+use common::Uint256;
+
+pub const TARGET_TIMESPAN_SECS: u32 = 14 * 24 * 60 * 60; // 2 weeks
+pub const TARGET_TIMESPAN_UINT256: Uint256 =
+    Uint256([0x0000000000127500, 0x0000000000000000, 0x0000000000000000, 0x0000000000000000]);
+
+pub const TARGET_SPACING: u32 = 10 * 60;
+pub const DIFFICULTY_ADJUSTMENT_INTERVAL: u32 = TARGET_TIMESPAN_SECS / TARGET_SPACING;
+
+/// taken from: https://github.com/bitcoin/bitcoin/blob/master/src/pow.cpp#L56
```

Comment:
Whenever you take a link from github, make sure you use a permalink, which contains the commit number. You can do this by clicking on the `...` on the left of the line number. Why? Because the master branch can always change.